var/spack/repos/builtin/packages/xrootd/package.py

```diff
@@ -39,3 +39,4 @@ class Xrootd(CMakePackage):
     version('4.3.0', '39c2fab9f632f35e12ff607ccaf9e16c')
 
     depends_on('cmake@2.6:', type='build')
+    depends_on('zlib')
```

Comment:
When you submit a PR, it's best practice to submit it on a branch, like `features/centos-rhel`, not on `develop`. Your PR can only contain a single change, whereas you may want to merge multiple changes into your `develop` branch. What I'm saying is, either redo this PR on a branch, or freeze the branch until this PR is merged (which may take a while).